charts/stable/twofauth/templates/_secrets.tpl

```diff
@@ -2,6 +2,12 @@
 {{- define "twofauth.secret" -}}
   {{- $secretName := (printf "%s-twofauth-secret" (include "tc.v1.common.lib.chart.names.fullname" $)) }}
 
+  {{ $URL := (printf "http://%s:%v/" .Values.config.nodeIP .Values.service.main.ports.main.port) }}
+
+  {{- if and (.Values.ingress.main.enabled) (gt (len .Values.ingress.main.hosts) 0) -}}
+    {{- $URL = (printf "https://%s/" (index .Values.ingress.main.hosts 0).host) -}}
+  {{- end -}}
+
```

Comment:
afaik no, this should use the common defined appurl, though its not documented very well atm.

Reply:
outdated commit.